### DISCUSSION TITLE
Berkeley QA Release ea6c7ee

### DISCUSSION THREAD

**Discussion by p-shahi** [MinaProtocol/mina#10947] [Berkeley]
Mina's ecosystem partner, O(1) Labs, is continuously working on improvements to the core protocol and today we are excited to announce an updated unstable release `ea6c7ee` that is ready to test on the Berkeley QANet! Please report any issues in the comments on this post or tag any issues you create with `berkeley-qa` and include berkeley `ea6c7ee` in the issue title.

### New Features/Improvements:
- Add a new graphql endpoint to query accounts for a token Id https://github.com/MinaProtocol/mina/pull/10927
- Pickles flexible width (making zk app provers more performant) https://github.com/MinaProtocol/mina/pull/10886
- Remove `zkapp_fee_payer` table from archive db https://github.com/MinaProtocol/mina/pull/10917
- Add genesis block, accounts at archive startup https://github.com/MinaProtocol/mina/pull/10913
- archive node fixes, new graphql endpoint to retrieve token accounts https://github.com/MinaProtocol/mina/issues/10738

Note: zkApps Test Transaction CLI tool cannot be used to create token accounts in this release.

### Fixes:
- Fixed archive node error when adding block https://github.com/MinaProtocol/mina/issues/10892

### Complete Changelog since `a4eae89` in:
Github Compare: https://github.com/MinaProtocol/mina/compare/a4eae892e1019f82708d34b12164b9f51465189e...ea6c7ee8596c0c6886d6a39d9507917a96eb382d

### Debian Packages:
**Debian Repository:**
First, set up and update the `unstable` Debian Repository for your platform. Replace the word `CODENAME` with the appropriate codename for your machine, one of `bionic`, `focal`, `stretch`, `buster`, or `bullseye` and run:
echo "deb [trusted=yes] http://packages.o1test.net/ CODENAME unstable" | sudo tee /etc/apt/sources.list.d/mina-unstable.list
sudo apt-get update
Then, install the package(s) that you need:
**Mina Daemon:** `sudo apt-get install -y mina-berkeley=1.3.1beta1-release-2.0.0-ea6c7ee`
**zkApps Test Transaction CLI:** `sudo apt-get install -y mina-zkapp-test-transaction=1.3.1beta1-release-2.0.0-ea6c7ee`
[ zkApps Transaction CLI Documentation](https://github.com/MinaProtocol/mina/blob/ecbb3d490dce0f199894ebdf787a03b8fec64bd4/src/app/zkapp_test_transaction/README.md)

### Docker Images:
Every docker image is built for each of the supported platforms, replace `CODENAME` in the images below with your choice of `bionic`, `focal`, `stretch`, `buster`, or `bullseye` to select the base docker image.
**Mina Daemon:** `minaprotocol/mina-daemon:1.3.1beta1-release-2.0.0-ea6c7ee-CODENAME-berkeley`

### Step by Step Guide:
This version will only connect and function properly on the newly launched Berkeley QA Network. *Docker only* Make sure to run with `--peer-list-url https://storage.googleapis.com/seed-lists/berkeley_seeds.txt`.

If you are running the correct version on the correct network **BERKELEY QA NET**, `mina client status` will show:
```
Chain id:     9eed0c40b1d9cbe4260c400cacb48335d664f2af34ff80de7211ffb36df6f2c9
Git SHA-1:    ea6c7ee8596c0c6886d6a39d9507917a96eb382d
```

**Comment by garethtdavies** [comment]
As posted in Discord. Archive schemas:

https://raw.githubusercontent.com/MinaProtocol/mina/ea6c7ee8596c0c6886d6a39d9507917a96eb382d/src/app/archive/create_schema.sql, https://raw.githubusercontent.com/MinaProtocol/mina/ea6c7ee8596c0c6886d6a39d9507917a96eb382d/src/app/archive/zkapp_tables.sql
